appcell-resources/js/settings.js

```diff
@@ -318,12 +318,24 @@ st.createAddAccount = function() {
     cm.setBackahead(true);
     cm.getRoleList().done(function(data) {
         var html = '<div class="modal-body">';
+        html += '<div id="dvAccType" data-i18n="AuthType"></div>';
```

Comment:
Please convert the "html += " to "$('<div>')" format so that it is easier to refactor.